dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/analytics/outlier/OutliersDetection2AutoTest.java

```diff
@@ -293,11 +290,11 @@ public void queryOutliertest7() throws JSONException {
         false);
 
     // Assert rows.
-    validateRow(response, 0, List.of("O05mAByOgAv", "OPV2 doses given", "98.5"));
-    validateRow(response, 1, List.of("vI2csg55S9C", "OPV3 doses given", "70.3"));
-    validateRow(response, 2, List.of("n6aMJNLdvep", "Penta3 doses given", "70.1"));
-    validateRow(response, 3, List.of("UOlfIjgN8X6", "Fully Immunized child", "59.8"));
-    validateRow(response, 4, List.of("I78gJm4KBo7", "Penta2 doses given", "59.0"));
+    validateRow(response, List.of("O05mAByOgAv", "OPV2 doses given", "98.48"));
```

Comment:
Is there a reason why you removed the indexes? If we have the `orderBy=modifiedzscore`, we should assert that the order is correct. I would keep the indexes. If the `orderBy` is not working it should be fixed...

Reply:
done